src/main/java/org/folio/persist/AbstractRepository.java

```diff
@@ -38,8 +38,8 @@ protected AbstractRepository(PostgresClient postgresClient, String tableName,
     this.recordType = recordType;
   }
 
-  public Future<String> save(String id, T entity) {
-    return postgresClient.save(tableName, id, entity);
+  public Future<T> save(String id, T entity) {
```

Comment:
Let's rename this method to saveAndReturnUpdatedEntity() to match postgresClient's method

Reply:
renamed